apps/parsley/src/hooks/useSections/index.ts

```diff
@@ -108,32 +114,39 @@ export const useSections = ({
     [],
   );
 
+  /**
+   * openSectionsContainingLineNumbers Will update the current section state. If the next state is the
+   * same as the current state the function will return true and false otherwise.
+   * @param param0 is an object with a lineNumber key(s).
+   * @param param0.lineNumbers is a number or an array of numbers that represent raw log line numbers.
```

Comment:
(maybe this is a good way to name the parameter)
```suggestion
   * @param options is an object with a lineNumber key(s).
   * @param options.lineNumbers is a number or an array of numbers that represent raw log line numbers.
```